src/britive/britive.py

```diff
@@ -180,7 +184,14 @@ def patch_upload(self, url, file_content_as_str, content_type, filename):
             return response.json()
         except native_json.decoder.JSONDecodeError:  # if we cannot decode json then the response isn't json
             return response.content.decode('utf-8')
-
+            
+    def post_upload(self, url, params=None, files=None):
+        """Internal use only."""
+        response = self.session.post(url, params=params, files=files, headers={'Content-Type': None})
+        try:
+            return response.json()
+        except native_json.decoder.JSONDecodeError:  # if we cannot decode json then the response isn't json
+            return response.content.decode('utf-8')
```

Comment:
space this out

Reply:
finished in 80810d7247c7a5e1e78a5765ab466861bb53a39c